test/files/neg/t6962.scala

```diff
@@ -0,0 +1,6 @@
+trait t6962 {
+  def sth() {
```

Comment:
Did you intend to avoid procedure syntax?

Reply:
I wanted to keep as close to the original example as possible. Fixed.